tools/online_editor/package.json

```diff
@@ -23,23 +24,24 @@
     "keywords": [],
     "author": "",
     "dependencies": {
-        "@lumino/default-theme": "^1.0.0-alpha.0",
-        "@lumino/dragdrop": "^2.0.0-alpha.0",
-        "@lumino/widgets": "^2.0.0-alpha.0",
-        "monaco-editor": "^0.34.0",
-        "monaco-languageclient": "^3.0.0",
+        "@lumino/default-theme": "^2.0.0-beta.1",
+        "@lumino/dragdrop": "^2.0.0-beta.1",
+        "@lumino/widgets": "^2.0.0-beta.1",
+        "monaco-editor": "^0.34.1",
+        "monaco-languageclient": "^4.0.3",
+        "path-browserify": "^1.0.1",
```

Comment:
Since this new dependency and all the others are bundled anyway, is there a reason to leave them in "dependencies" instead of having them all in "devDependencies"?